scripts/packages/VSCodeServer/src/display.jl

```diff
@@ -82,6 +82,33 @@ const DISPLAYABLE_MIMES = [
     "image/gif"
 ]
 
+"""
+    DIAGNOSTIC_MIME = "application/vnd.julia-vscode.diagnostics"
+
+Trace format is:
+```
+[
+    Dict(
+        "msg" => ""::AbstractString,
+        "path" => ""::AbstractString,
+        "line" => 0::Int
+    ),
+    ...
+]
+```
+or everything JSON-compatible with that (so e.g. a struct with those fields).
```

Comment:
I mean, we can change the docs so that only named tuples or dicts or whatever work; the implementation basically prescribes "if JSON and only JSON is loaded, any object that JSON-serializes to something of that format works". I don't see any benefit to checking this anywhere.